agent/api/ecsclient/client.go

```diff
@@ -157,7 +157,9 @@ func (client *APIECSClient) registerContainerInstance(clusterRef string, contain
 	// Add additional attributes such as the os type
 	registrationAttributes = append(registrationAttributes, client.getAdditionalAttributes()...)
 	registerRequest.Attributes = registrationAttributes
-	registerRequest.Tags = tags
+	if tagsEnabled && len(tags) > 0 {
```

Comment:
i don't think we need the `tagsEnabled` parameter. we can add an else block here for setting this field, lets try to keep the interfaces smaller if we can help it.

Reply:
Just removed the parameter